addons/web/static/tests/_framework/env_test_helpers.js

```diff
@@ -116,14 +116,16 @@ export async function makeMockEnv(partialEnv, { makeNew = false } = {}) {
  * Makes a mock environment for dialog tests
  *
  * @param {Partial<OdooEnv>} [partialEnv]
+ * @param {Partial<OdooEnv>} [dialogData]
  * @returns {Promise<OdooEnv>}
  */
-export async function makeDialogMockEnv(partialEnv) {
+export async function makeDialogMockEnv(partialEnv, dialogData) {
     return makeMockEnv({
         dialogData: {
             close: () => {},
             isActive: true,
             scrollToOrigin: () => {},
+            ...dialogData,
```

Comment:
`dialogData` can be useless

e.g.:
```javascript
makeDialogMockEnv({test: "test1", dialogData: {test: "test2"}}, {test: "test3"});
```
`dialogData` param will be override by `partialEnv.dialogData`

The following example can be a solution but create another problem
```javascript
function makeDialogMockEnvMod(partialEnv) {
    return makeMockEnv({
        ...partialEnv,
        dialogData: {
            close: () => {},
            isActive: true,
            scrollToOrigin: () => {},
            ...partialEnv?.dialogData,
        },
    });
}
```
we can't reset `env.dialogData` (`partialEnv.dialogData`) anymore.